### PR TITLE
Feature/add coverage

### PR DESCRIPTION
**Pull request by stewartie4**

### CODE REVIEW

File: tests/cli_tests/zboxcli_transfer_allocation_test.go [hunk 1]
```diff
@@ -54,6 +56,117 @@ func TestTransferAllocation(t *testing.T) { // nolint:gocyclo // team preference
...
+		require.Nil(t, err, strings.Join(output, "\n"))
+		require.Len(t, output, 1, "transfer allocation - Unexpected output", strings.Join(output, "\n"))
+		require.Equal(t, fmt.Sprintf("transferred ownership of allocation %s to %s", allocationID, newOwnerWallet.ClientID), output[0],
+			"transfer allocation - Unexpected output", strings.Join(output, "\n"))
+
+		transferred := pollForAllocationTransferToEffect(t, newOwner, allocationID)
+		require.True(t, transferred, "allocation was not transferred to new owner within time allotted")
+
+		// balance of old owner should be unchanged
+		// FIXME should this contain the released pool balances given the change of ownership?
```
Comment (stewartie4) [inline tests/cli_tests/zboxcli_transfer_allocation_test.go:133]: @dabasov do you know if this is still expected?

File: tests/cli_tests/zboxcli_transfer_allocation_test.go [hunk 2]
```diff
@@ -54,6 +56,117 @@ func TestTransferAllocation(t *testing.T) { // nolint:gocyclo // team preference
...
+		require.Nil(t, err, "Unexpected balance check failure for wallet", escapedTestName(t), strings.Join(output, "\n"))
+		require.Len(t, output, 1, "get balance - Unexpected output", strings.Join(output, "\n"))
+		require.Regexp(t, regexp.MustCompile(`Balance: 1.000 ZCN \(\d*\.?\d+ USD\)$`), output[0],
+			"get balance - Unexpected output", strings.Join(output, "\n"))
+
+		// zero cost to transfer
+		expectedTransferCost := int64(0)
+
+		// write lock pool of old owner should remain locked
+		// FIXME should this be unlocked given the change of ownership?
```
Comment (stewartie4) [inline tests/cli_tests/zboxcli_transfer_allocation_test.go:151]: @dabasov do you know if this is still expected?

File: tests/cli_tests/zboxcli_transfer_allocation_test.go (hunk 1 above)
Comment (dabasov) [reply tests/cli_tests/zboxcli_transfer_allocation_test.go:133]: we do not release any pools here, simply remove allocation from old owner list and add it to the new owner list

Comment (dabasov) [reply tests/cli_tests/zboxcli_transfer_allocation_test.go:133]: so balance stays unchanged now

File: tests/cli_tests/zboxcli_transfer_allocation_test.go (hunk 2 above)
Comment (dabasov) [reply tests/cli_tests/zboxcli_transfer_allocation_test.go:151]: no change to balance atm

**Review by dabasov — APPROVED** [review]
lgtm

File: tests/cli_tests/zboxcli_transfer_allocation_test.go (hunk 2 above)
Comment (stewartie4) [reply tests/cli_tests/zboxcli_transfer_allocation_test.go:151]: thanks Yury

**Review by Kishan-Dhakan — COMMENTED** [review]
LGTM

**Pull request merged**